### PR TITLE
Point NPM module to transpiled distribution

### PR DESCRIPTION
**Pull request by josephschmitt**
Bumps the version to 0.0.2. Also adds angular as an external dependency so it's not included in the main bundle. Updated the Example Apps so they load in angular separately.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-  "main": "src/index.js",
+  "version": "0.0.2",
+  "description": "A modern, flexible, Angular 1.x autocomplete library with limited assumptions.",
+  "main": "dist/ngc-omnibox.js",
   "scripts": {
     "bundle": "webpack --progress --colors",
     "dev": "webpack --progress --colors --watch",
-    "test": "istanbul cover --include-all-sources babel-node node_modules/.bin/jasmine && eslint ."
+    "test": "istanbul cover --include-all-sources babel-node node_modules/.bin/jasmine && eslint .",
+    "prepublish": "npm run build"
```
Comment (onlywei) [inline package.json]: I don't see a "build" script here in package.json. There's a "bundle" script.. am I missing something?

File: .gitignore [hunk 2]
```diff
@@ -11,7 +11,6 @@ dist/
 
 # Dependency directories
 node_modules
-jspm_packages
```
Comment (onlywei) [inline .gitignore]: why?

Comment (josephschmitt) [inline .gitignore]: We don't have a jspm_packages folder. This was there just because I had copy/pasted from our codebase.

Comment (onlywei) [inline .gitignore]: oh right

**Pull request merged**